README.md

```diff
@@ -31,9 +31,23 @@ scoop bucket add svix https://github.com/svix/scoop-svix.git
 scoop install svix
 ```
 
-### Other
+### Linux
 
-If you are on another platform such as linux or just rather not use a package manager, you can download a binary from our Github releases and use it straight away without having to install any additional dependencies.
+The Svix CLI is available on Linux via the [snap](https://snapcraft.io/) package manager:
+
+```sh
+snap install svix
+```
+
+And via the [Arch User Repository (AUR)](https://wiki.archlinux.org/title/Arch_User_Repository):
+
+```sh
+pacman -S svix-cli
```

Comment:
@tasn please confirm that that is the correct pacman command, its been a while since I've used arch 😅

Reply:
It's the correct pacman command, but since it's in AUR it's not installed by pacman, but rather by tools like `yay`.

```suggestion
yay -S svix-cli
```